index.js

```diff
@@ -109,6 +109,7 @@ function buildAssets (opts, callback) {
             }
             rawScss.push('@import "'+opts.variables+'";');
         }
+        rawScss.push('@import "'+getModulePath('slate')+'/source/stylesheets/_variables.scss";');
```

Comment:
No. From the SASS documentation:

> You can assign to variables if they aren’t already assigned by adding the !default flag to the end of the value. This means that if the variable has already been assigned to, it won’t be re-assigned, but if it doesn’t have a value yet, it will be given one.

So, if you specify a variable it will be used and if you don't specify a variable then the Slate variable will be used. I have tested with the order in the pull request (manually changing the Slate variables to !default) and it works.